.drone/drone.jsonnet

```diff
@@ -3,6 +3,8 @@ local archs = ['amd64', 'arm64', 'arm'];
 
 local build_image_version = std.extVar('__build-image-version');
 
+local drone_updater_plugin_image = 'us.gcr.io/kubernetes-dev/drone/plugins/updater@sha256:cbcb09c74f96a34c528f52bf9b4815a036b11fed65f685be216e0c8b8e84285b';
```

Comment:
Is this a drone plugin or a specific one for Grafana? 
nit: Should we maybe point to main?

Reply:
it's drone plugin that we developed https://github.com/grafana/deployment_tools/tree/master/drone/plugins/cmd/updater 

the release process of this plugin provides only `latest` version of the image. so, we decided to use image `sha` to stick to needed version ;)